core/src/main/java/io/questdb/cairo/TxnScoreboard.java

```diff
@@ -65,9 +65,10 @@ public boolean acquireTxn(long txn) {
         throw CairoException.instance(0).put("max txn-inflight limit reached [txn=").put(txn).put(", min=").put(min).put(", size=").put(pow2EntryCount).put(']');
     }
 
-    public void releaseTxn(long txn) {
+    public long releaseTxn(long txn) {
```

Comment:
the return value is always used in `boolean` context or not at all

Reply:
I think it's cleaner API to return number of locks after release, like atomics do

Reply:
this is minor and subjective, i'm ok with it 👍